release/__init__.py

```diff
@@ -79,12 +75,8 @@ def strip_locals(data):
     """
 
     if isinstance(data, dict):
-        data = copy.copy(data)
-        for k in set(data.keys()):
-            if isinstance(k, str) and k.startswith('local_'):
-                del data[k]
-            else:
-                data[k] = strip_locals(data[k])
+        return {key: strip_locals(value) for key, value in data.items()
+                if not (isinstance(k, str) and k.startswith('local_'))}
```

Comment:
k needs to be changed to key. If you run `tox -e py34-unittests` to run the tests appliacble to this file, a python NameError exception will get thrown here 

(I'm currently working on docs for how to run code quality checks among other things)